components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx

```diff
@@ -22,15 +22,15 @@ export const MultipleChoiceGroup = (props: MultipleChoiceGroupProps): React.Reac
         key={index}
         name={field.name}
         className={className}
-        ariaDescribedBy={ariaDescribedBy}
+        {...(ariaDescribedBy && { ariaDescribedBy: ariaDescribedBy })}
```

Comment:
Should we just remove this ?

I don't see any code using the ariaDescribedBy for MultipleChoiceGroup

Reply:
Sure. On second thought, leaving that there may cause more harm than good.